src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,43 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 24px;
+
+  @include on744px {
+    grid-template-columns: repeat(6, 1fr);
+  }
+
+  @include on320px {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  &__item {
+    @for $start from 1 through 12 {
+      @for $end from $start through 12 {
+        &--#{$start}-#{$end} {
+          grid-column: #{$start} / #{$end + 1};
+        }
+      }
+    }
+
+    @include on744px {
+      @for $start from 1 through 6 {
+        @for $end from $start through 6 {
+          &--on744px-#{$start}-#{$end} {
```

Comment:
Does this modifier mean that on 745 or 743px this style doesn't work? 🤔

Reply:
this modifier divides the page into columns depending on the screen size